Applications/SimpleFC/jni/main.cpp

```diff
@@ -155,25 +132,13 @@ void createAndRun(unsigned int epochs, unsigned int batch_size,
     throw std::invalid_argument("model initialization failed!");
   }
 
-  auto dataset_train = ml::train::createDataset(
-    ml::train::DatasetType::GENERATOR, trainData_cb, train_user_data.get());
-  auto dataset_valid = ml::train::createDataset(
-    ml::train::DatasetType::GENERATOR, validData_cb, valid_user_data.get());
-
-  // to test asynch fsu, we do need save the model weight data in file
-  model->save("simplefc_weight_fp16_fp16_100.bin",
-              ml::train::ModelFormat::MODEL_FORMAT_BIN);
-  model->load("./simplefc_weight_fp16_fp16_100.bin");
-
-  model->summarize(std::cout, ML_TRAIN_SUMMARY_MODEL);
-
-  unsigned int feature_size = 320;
+  uint feature_size = 1 * 1024 * 1440;
```

Comment:
let's not use `uint` as in #2890
```suggestion
  unsigned int feature_size = 1 * 1024 * 1440;
```